
Merges what really makes sense from the Chris Wright patch. Most of the other
changes are either already merged or redundant. And some symlinks are already
cleaned with make *mrproper*.

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
---

 linux-2.6.9-current-paolo/arch/um/Makefile |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/um/Makefile~uml-kbuild-add-even-more-cleaning arch/um/Makefile
--- linux-2.6.9-current/arch/um/Makefile~uml-kbuild-add-even-more-cleaning	2004-09-19 19:59:41.177611496 +0200
+++ linux-2.6.9-current-paolo/arch/um/Makefile	2004-09-19 20:28:34.259143056 +0200
@@ -123,7 +123,7 @@ CLEAN_FILES += linux x.i gmon.out $(ARCH
 	$(GEN_HEADERS) $(ARCH_DIR)/include/skas_ptregs.h
 
 MRPROPER_FILES += $(SYMLINK_HEADERS) $(ARCH_SYMLINKS) \
-	$(addprefix $(ARCH_DIR)/kernel/,$(KERN_SYMLINKS))
+	$(addprefix $(ARCH_DIR)/kernel/,$(KERN_SYMLINKS)) $(ARCH_DIR)/os
 
 archmrproper:
 	@:
_
